container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -3258,218 +3099,29 @@ variables:
     description: A PEM-encoded TLS key of the Autoscaler Metrics https server. This
       includes the Autoscaler Metrics Collector.
     required: true
-- name: AUTOSCALER_DATABASE_MAX_CONNECTIONS
+- name: AUTOSCALER_DB_ADDRESS
   options:
-    default: 1000
-    description: The maximum number of connections the Autoscaler postgres database
-      server could serve.
+    default: "autoscaler-postgres-postgres.((KUBERNETES_NAMESPACE)).svc.((KUBERNETES_CLUSTER_DOMAIN))"
+    description: The password for the Autoscaler postgres database.
+    required: true
```

Comment:
The `descriptions` sounds wrong.

And `required: true` is meaningless when you have a `default` value.